modules/appeals_api/app/swagger/higher_level_reviews/v0/swagger.json

```diff
@@ -793,17 +813,37 @@
               }
             }
           },
-          "404": {
-            "description": "Higher-Level Review record was not found, or the provided 'icn' query parameter does not match the record's ICN",
+          "400": {
+            "description": "Missing 'icn' query parameter (with a system- or representative-scoped OAuth token)",
             "content": {
               "application/json": {
                 "example": {
                   "errors": [
                     {
-                      "title": "Resource not found",
-                      "detail": "Higher-Level Review with uuid 44444444-5555-6666-7777-888888888888 not found",
-                      "code": "404",
-                      "status": "404"
+                      "title": "Missing parameter",
+                      "detail": "The 'icn' parameter is required with this request",
+                      "code": "108",
+                      "status": "400"
+                    }
+                  ]
+                },
+                "schema": {
+                  "$ref": "#/components/schemas/errorModel"
+                }
+              }
+            }
+          },
+          "403": {
```

Comment:
Initially, I started to write several more docs specs for other scopes in these APIs - examples with system tokens, representatives, veterans, etc. - but I realized that the UI does not actually render curl requests for each example - it only does that once for each endpoint, and all of the example responses just show the body that's returned - not the scopes or parameters that were sent. So I kept the number of example errors smaller and tried to communicate about scopes and params using the `description` fields instead.